### PR TITLE
Feature/update link in documentation

### PR DESCRIPTION
**Pull request by maxime-urbanski**
* `fix`: Fix path for CONTRIBUTING.md

### CODE REVIEW

File: docs/hub/install.md [hunk 1]
```diff
@@ -40,7 +40,7 @@ When the server is up and running, the following endpoints are available:
 
 See [the protocol](spec/mercure.md) for more details about these endpoints.
 
-To compile the development version and register the demo page, see [https://github.com/dunglas/mercure/blob/master/CONTRIBUTING.md](CONTRIBUTING.md#hub).
+To compile the development version and register the demo page, see [https://github.com/dunglas/mercure/blob/master/CONTRIBUTING.md](../../CONTRIBUTING.md#hub).
```
Comment (dunglas) [inline docs/hub/install.md:43]:
```suggestion
To compile the development version and register the demo page, see [https://github.com/dunglas/mercure/blob/master/CONTRIBUTING.md](https://github.com/dunglas/mercure/blob/master/CONTRIBUTING.md).
```

**Review by dunglas — CHANGES_REQUESTED** [review]
Or the link will break on the website.

File: docs/hub/install.md (hunk 1 above)
Comment (maxime-urbanski) [reply docs/hub/install.md]: Ok, i changed the link.

**Comment by dunglas** [comment]
Thanks @maxime-urbanski

**Pull request merged**